examples/hosts/app-integration/external-data/tests/index.tsx

```diff
@@ -9,9 +9,28 @@ import ReactDOM from "react-dom";
 import { SessionStorageModelLoader, StaticCodeLoader } from "@fluid-example/example-utils";
 
 import { TaskListContainerRuntimeFactory } from "../src/model";
-import type { IAppModel } from "../src/model-interface";
+import type { IAppModel, ITaskList } from "../src/model-interface";
 import { TaskListView } from "../src/view";
 
+/**
+ * The prop for the TestView FC wrapper. Used to pass tasklist to TaskListView.
+ */
+export interface ITestViewProps {
+	/**
+	 * The Task List app model to be visualized.
+	 */
+	taskList: ITaskList;
+}
+/**
+ * This is a functional component wrapper for TaskList view, which is needed to use and pass in state.
+ * @param props -the taskList prop used by the TaskListView component.
+ * @returns TaskListComponent with React hooks passed in.
+ */
+export const TestView: React.FC<ITestViewProps> = (props: ITestViewProps) => {
+	const { taskList } = props;
+	return <TaskListView taskList={taskList} />;
+};
```

Comment:
Help the React newbie here 😄. What's the purpose of this component? Seems to just proxy `TaskListView` with no additional functionality.

Reply:
I had added this when I was coupling the Debug and TaskList under than same component so they could share state. Will remove!